pkg/el/google-fluentd-catch-all-config.spec

```diff
@@ -0,0 +1,40 @@
+Summary: Catch-all configuration files for google-fluentd on GCE.
+Name: google-fluentd-catch-all-config
+Version: 0.1
+Release: 1%{?dist}
+License: Apache
```

Comment:
ASL 2.0 to match LICENSE (I did see that page before, but elsewhere I saw conflicting information about which format to use).